cairo/ethereum/cancun/state.cairo

```diff
@@ -937,6 +936,30 @@ func destroy_touched_empty_accounts{poseidon_ptr: PoseidonBuiltin*, state: State
     );
 }
 
+func empty_transient_storage{range_check_ptr}() -> TransientStorage {
+    tempvar default_value = new U256Struct(0, 0);
+    let (dict_ptr) = default_dict_new(cast(default_value, felt));
```

Comment:
The default value is a pointer to a U256(0, 0)
How will this translate when calling default_dict_finalize?
How to retrieve this pointer to pass it as the default value?

Reply:
oh you would have create a label and use get_label_location like @enitrat did storage root right ?

Reply:
@Eikix so here you need to match something like this https://github.com/kkrt-labs/keth/blob/5d4d479ef2f0fc38cc7a47587f7f52835de33d3e/cairo/ethereum/cancun/state.cairo#L992

Reply:
actionable:
cairo.legacy.utils.uint256.cairo:
```
U256_ZERO:
dw 0;  // low
dw 0;  // high
```

[cairo/ethereum/cancun/state.cairo](https://github.com/kkrt-labs/keth/pull/769/files#diff-7127136c68c270e3fc6450495d32ac5de0f7982b038507158ec34402c5acb27c)
:

```
from starkware.cairo.common.registers import get_label_location
from cairo.legacy.utils.uint256 import U256_ZERO
...

let (u256_zero) = get_label_location(); 
let (dict_ptr) = default_dict_new(cast(u256_zero, felt));

Reply:
@Eikix ?